CoreFoundation/RunLoop.subproj/CFSocket.c

```diff
@@ -2174,9 +2175,9 @@ manageSelectError()
 
 static void *__CFSocketManager(void * arg)
 {
-#if (TARGET_OS_LINUX && !TARGET_OS_CYGWIN) || TARGET_OS_BSD
+#if (TARGET_OS_LINUX && !TARGET_OS_CYGWIN)
```

Comment:
Would you mind dropping the parenthesis?

Reply:
Done.